packages/ckeditor5-widget/src/widget.ts

```diff
@@ -249,6 +244,38 @@ export default class Widget extends Plugin {
 		this._setSelectionOverElement( modelElement! );
 	}
 
+	/**
+	 * Selects entire block content, e.g. on triple click it selects entire paragraph.
+	 */
+	private _selectBlockContent( element: ViewElement ): boolean {
```

Comment:
This code could be simpler, we could use some existing helper methods. Let's talk about it f2f.